test/terra/backends/test_qasm_simulator_matrix_product_state.py

```diff
@@ -71,6 +72,7 @@ class TestQasmMatrixProductStateSimulator(
         QasmPauliNoiseTests,
         QasmResetNoiseTests,
         QasmKrausNoiseTests,
+        QasmKrausNoiseOnQFTTests,
```

Comment:
Why not test the other simulation methods with the new test?

Reply:
I wasn't sure it is worthwhile to check the other methods. I will check which methods I can add.